internal/kafka/test/integration/kafkas_test.go

```diff
@@ -804,15 +797,15 @@ func TestKafkaCreate_TooManyKafkas(t *testing.T) {
 		resp.Body.Close()
 	}
 
-	g.Expect(err).ToNot(gomega.HaveOccurred(), "Ung.Expected error occurred when creating kafka:  %v", err)
+	g.Expect(err).ToNot(gomega.HaveOccurred(), "Expected error occurred when creating kafka:  %v", err)
```

Comment:
This should be `Unexpected` instead of `Expected`

Reply:
good catch.